Explorer/Assets/DCL/PluginSystem/Global/MultiplayerPlugin.cs

```diff
@@ -100,21 +105,23 @@ public MultiplayerPlugin(
             this.poolsRegistry = poolsRegistry;
         }
 
+        public void Dispose() { }
```

Comment:
Does debugRoomIndicatorPool need disposing here?

Reply:
Usually it's called on application destroy, so we don't care. We rarely dispose of pools explicitly - its not really needed.